CONTRIBUTING.md

```diff
@@ -34,64 +24,93 @@ The services are deployed as the same docker `arkhn/river` image, but are run wi
 
-You must provide `fhir-api` and `pyrog-api` URLs (which means you have them deployed somewhere)
-
-#### `docker-compose.monitoring.yml`
-
-Contains optional monitoring services.
+### Notes on environment
 
-The services config files are in the root `monitoring` directory.
+- Environment variables can be stored in a `.env` file in the root directory.
+- The `.env` file is shared and loaded by:
+  - `docker-compose`,
+  - Django's `manage.py` script,
+  - Vscode's integrated terminal and launch commands.
 
-#### `docker-compose.test.yml`
+## Backend development
 
-Optional test services.
+### Local prerequisites
 
-### Notes on environment
+```bash
+# 1. Create a virtual env
+python3.8 -m venv --prompt "river" .venv
 
-- Environment variables can be stored in a `.env` file in the root directory.
-- The `.env` file is loaded by:
-  - Django's `manage.py` script, in development,
-  - Vscode's integrated terminal and launch commands,
-  - Docker-compose's cli.
+# 2. Activate the virtual env
+source .venv/bin/activate
 
-## Local development
+# 3. Install dev requirements
+pip install -r requirements/dev.txt
+```
 
-### Local prerequisites
+### Developing with the development server
 
-1. Create a virtual env
+This concerns the API. Not the ETL services (which are not web applications).
 
-   python3.8 -m venv --prompt "river" .venv
+First, you'll need to provide configuration by creating a dotenv file (`.env`).
 
-2. Activate the virtual env
+```bash
+# 1. Copy the dotenv file. The template should be enough to get you started.
+cp .template.env .env
 
-   source .venv/bin/activate
+# 2. Bring the db up
+docker-compose up -d db
 
-3. Install dev requirements
+# 3. Migrate
+python django/manage.py migrate
 
-   pip install -r requirements/dev.txt
+# 4. Run the development server
+python django/manage.py runserver
+```
 
 ### Code quality
 
 Code quality is enforced with `pre-commit` hooks: `black`, `isort`, `flake8`
 
-1. Install the hooks
-
-   pre-commit install
-
+```bash
+# Install the hooks
+pre-commit install
+```
 ### Tests
 
-1.  Run tests in dedicated virtual env.
-
-        tox
+```bash
+# Run tests in dedicated virtual env
+tox
+```
 
-## OpenAPI schema generation
+### OpenAPI schema generation
 
 Generation only concerns the `pyrog` application at this point. To generate the schema (no virtual env required):
 
       tox -e openapi
 
 This produces a `pyrog-schema.yml` file in the root project directory.
+
+## Frontend development
+
+### Backend setup
+
+First, you'll need to provide configuration by creating a dotenv file (`.env`).
+
+```bash
+# 1. Copy the dotenv file. The template should be enough to get you started.
+cp .template.env .env
+
+# 2. Bring the db up
+docker-compose up -d db
+
+# 3. Bring the backend up
+docker-compose up river-api
+
+# 4. (Optionally) To quickly create resources, visit the admin panel
```

Comment:
I guess this is protected by a password, can you mention that users may use
```
id: admin
pwd: admin
```
in development (as specified in the docker-compose) ?